README.md

```diff
@@ -1,10 +1,12 @@
-# Google Calendar -> Twitch Schedule Mirror
+# Google Calendar -> Twitch & Discord Schedule Mirror
 
-Update a Twitch schedule based on a Google calendar link.
+Update a Twitch schedule and add Discord events based on a Google calendar link.
 
 This script will:
 * Remove all upcoming events on your Twitch Schedule
 * Create an upcoming event on your Twitch Schedule for every upcoming event on the given Google calendar
+* Find Discord events that have `[CREATION_TIMESTAMP]` in their description and edit them according to the Google calendar
```

Comment:
Update `[CREATION_TIMESTAMP]` to be `[EVENT UID]`

Reply:
Done.